src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -610,13 +593,12 @@ public T AddKnownAdfsAuthority(string authorityUri, bool isDefaultAuthority)
         /// </summary>
         /// <param name="authorityUri">Azure AD B2C authority, including the B2C policy (for instance
         /// <c>"https://fabrikamb2c.b2clogin.com/tfp/{Tenant}/{policy}</c></param>)
-        /// <param name="isDefaultAuthority">Boolean telling if this is the default authority
-        /// for the application</param>
+        /// <param name="validateAuthority">TODO(migration): documentation</param>
         /// <remarks>You can add several authorities, but only one can be the default authority.</remarks>
         /// <returns>The builder to chain the .With methods</returns>
-        public T AddKnownB2CAuthority(string authorityUri, bool isDefaultAuthority)
+        public T WithB2CAuthority(string authorityUri, bool validateAuthority)
```

Comment:
just a thought....b2c users should always pass validateAuthority=false. 
can we have b2c devs pass in only the authorityUri, and we set validateAuthority to false for them? just one less thing for them to think about and less documentation for b2c when using msal.

Reply:
Yup, will do.

Reply:
Then a todo will be to let b2c know so they can update their docs. :) I'll add this to the onenote somewhere